index.html

```diff
@@ -14,53 +15,15 @@
 
 <body>
     <header>
-        <div class="gutter-right logo">
-            <img src="./assets/big-logo.png" />
-        </div>
+      <button id="loginbtn">Login or Sign up</button>
+      <button id="createbtn">Create</button>
```

Comment:
We can dynamically render login versus logout by accessing the textContent of the button. Using a getUser function, we can determine if the user exists which would render logout functionality or login functionality if there is no user.